src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -52,7 +54,7 @@
 
         this.shouldRedirectExplicitDefaultLangUrl = this.urlLanguagePatternHandler.shouldRedirectExplicitDefaultLangUrl(clientRequestUrl);
 
-        this.isValidRequest = this.requestLang != null && this.urlContext != null;
+        this.isValidRequest = this.requestLang != null && this.urlContext != null && currentContextUrlIgnoredPath == false;
```

Comment:
`UrlContext` shouldn't really care about ignore paths. It would be better as a method in headers:
`this.isIgnoredPath(this.urlContext.getPath())`